src/dappkit/DappKitAccountScreen.tsx

```diff
@@ -28,6 +30,9 @@ type Props = BottomSheetScreenProps<StackParamList, Screens.DappKitAccountScreen
 const DappKitAccountScreen = ({ route, handleContentLayout }: Props) => {
   const { dappKitRequest } = route.params
 
+  const [scrollEnabled, setScrollEnabled] = useState(false)
+  const { height } = useSafeAreaFrame()
```

Comment:
Shouldn't we include the top area height with the notch?

Otherwise it goes too high from the video taken:
<img width="379" alt="Screenshot 2023-02-07 at 19 12 29" src="https://user-images.githubusercontent.com/57791/217330698-854318e8-0c3e-4a8f-a7f1-18acb2d9f287.png">

Reply:
yes you're right! i accounted for the notch now when scrolling, but ideally i wanted the bottom sheet handle not to overlap with the status bar / notch. i couldn't figure out a way to do this with dynamic snap points, it seems like this is the default behaviour when you set the snap point to 100% :( 

updated scrolling:

https://user-images.githubusercontent.com/20150449/217485829-693b1780-9442-4d57-9944-f575f14a882f.mp4


https://user-images.githubusercontent.com/20150449/217485842-2a988487-9748-4fbb-afda-35b95c6091ee.mp4

Reply:
Great! Nothing to do now, but if we wanted the top of the sheet to stop just before the top safe area, we could calculate the max percentage to use for the snap point. Knowing 100% is fullscreen and the height of the window + top inset.

Reply:
can you elaborate on how set the max percentage for the snap point? this is the part i am stuck on, since we just define a dynamic snap point and the `useBottomSheetDynamicSnapPoints` sets the actual value. i'd love to cap the maximum snap point, so keen if you have any ideas.

for now i'll merge this to not block the release but i'll follow up if we can limit the top of the sheet!